### PR TITLE
Build 3.0.2 for Windows

### PR DESCRIPTION
**Pull request by gillins**
This is WIP.

I'll update to version 3.0.2 when I get a moment.

See https://github.com/conda-forge/qgis-feedstock/issues/17. Note there are some issues on Linux so ignoring that for now.

### CODE REVIEW

**Comment by gillins** [comment]
@conda-forge-admin, please rerender

**Comment by gillins** [comment]
@conda-forge-admin, please rerender

**Comment by gillins** [comment]
Any ideas why OSX build failed with Circle CI (https://circleci.com/gh/conda-forge/qgis-feedstock/58?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) but got much further with Travis CI before timing out (https://travis-ci.org/conda-forge/qgis-feedstock/jobs/377084145)?

@ocefpaf has this been seen before?

File: recipe/build.sh [hunk 1]
```diff
@@ -9,26 +9,56 @@
...
+ninja install
+
+if [ $(uname) == Darwin ]; then
+  # rename it back
+  mv $PREFIX/include/uuid/uuid.h.bak $PREFIX/include/uuid/uuid.h
+
+  # also create this dir or creating the conda package failes due to broken link
+  mkdir -p $PREFIX/QGIS.app/Contents/MacOS/share
+
+  # and create a link into the .app so we can run it.
```
Comment (gillins) [inline recipe/build.sh]: Does anyone know how to convert an OSX `.app` back into the normal Unix directory structure? Is there already a script to do this?

**Comment by isuruf** [comment]
Looks like a bug with new clang/xcode and qscintilla. https://www.riverbankcomputing.com/pipermail/qscintilla/2016-October/001160.html

**Comment by gillins** [comment]
Thanks @isuruf

**Comment by gillins** [comment]
I think this one is ready to go. I've dropped OSX until we can get it building on the CIs (and work out the `.app` issue).

**Pull request merged**